gradle.properties

```diff
@@ -1,3 +1,21 @@
+GROUP=app.cash.duktape
```

Comment:
Intentional change?
```suggestion
GROUP=com.squareup.duktape
```

Reply:
Hmm yeah let's leave duktape in the old and quickjs in the new